phpBB/phpbb/textformatter/s9e/parser.php

```diff
@@ -56,24 +50,20 @@ public function __construct(\phpbb\cache\driver\driver_interface $cache, $key, \
 
 		$this->dispatcher = $dispatcher;
 		$this->parser = $parser;
-		$this->user = $user;
 		$parser = $this;
 
 		/**
 		* Configure the parser service
 		*
 		* Can be used to:
-		*  - toggle features according to the user's preferences,
-		*  - toggle BBCodes according to the user's permissions,
```

Comment:
Ok, then I officially declare this PR not-WIP anymore. 3.2 has the same functionality as 3.1. @Nicofuma you can merge if you want.